src/components/Renderer/RendererPlayground/util.ts

```diff
@@ -53,8 +52,8 @@ const generateArrayMetaSchema = (
 	{ value, schema, uiSchema, extraContext, extraFormats }: MetaSchemaArgs,
 ) => {
 	const arrayValue = value as Value[];
-	const itemsUiSchema = get(uiSchema, 'items', {});
-	const itemsSchema = get(schema, 'items', {});
+	const itemsUiSchema = uiSchema?.items ?? {};
+	const itemsSchema = (schema.items as JSONSchema) ?? {};
```

Comment:
There are casts like this all over the place which were previously hidden behind the `get`, since `get` ends up casting to `any` in most places. These explicit casts may look (and are imo) ugly/bad but they're just exposing issues that already exist in the code so we can no longer ignore them as easily